setup/includes/test/modinstalltest.class.php

```diff
@@ -66,8 +66,8 @@ protected function _checkPHPVersion()
         $this->title('php_version', $this->install->lexicon('test_php_version_start') . ' ');
         $phpVersion = phpversion();
 
-        $recommended_version = "7.3";
-        $required_version = "7.2.5";
+        $recommended_version = "8.0.0";
```

Comment:
At this stage, PHP 8.0.x is no longer supported. Should this be bumped to 8.1.0 (supported through Nov 25, 2024)?

Reply:
I’m no longer involved in this project so I don’t know. Maybe @opengeek or @Mark-H can help decide this.